### ISSUE TITLE
Game crash when i click or scroll on an object in all menu

### ISSUE THREAD

**Issue by Pari999** [Arkensor/DayZCommunityOfflineMode#382]
When i am on the all menu for spawing object and i click or scroll on the object "Land_VASICore" the game crash.

**Comment by Arkensor** [comment]
Yeah, those are probably illegal objects to spawn. Are they from vanilla? if so I can blacklist them, but the class name looks modded.

**Comment by Pari999** [comment]
> Yeah, those are probably illegal objects to spawn. Are they from vanilla? if so I can blacklist them, but the class name looks modded.

i think it is vanilla

**Comment by kamiisk** [comment]
I'm also experiencing crashes in the game when I hover over the item 'Land_VASICore'. I tried to quickly search the internet, and it seems to be some kind of legacy thing from Arma 3. Is there any way to put this item on a blacklist, so it won't be loaded into the game and cause crashes every time? Also, if I encounter other problematic items, is there a way to fix them? It's quite frustrating when the mod simply closes unexpectedly.

**Comment by kamiisk** [comment]
I managed to stop the crashing by adding the item to the protected autoprt matrix within the ObjectMenu.c file (Missions\DayZCommunityOfflineMode.ChernarusPlus\core\modules\ComEditor\gui) manually, as discussed in this topic:
https://github.com/Arkensor/DayZCommunityOfflineMode/issues/311#issuecomment-1001046083

**Comment by Anyone8111** [comment]
i'm getting the same issue, it's definitely vanilla

**Comment by Andydeluxe97** [comment]
> Ja, das sind wahrscheinlich illegale Objekte zu laichen. Sind sie von Vanille? Wenn ich sie also auf die schwarze Liste setzen kann, aber der Klassenname sieht um sich modded aus.

Yes this is from Vanilla, I think it's just the one where the game crashes -.-

**Comment by Andydeluxe97** [comment]
> I managed to stop the crashing by adding the item to the protected autoprt matrix within the ObjectMenu.c file (Missions\DayZCommunityOfflineMode.ChernarusPlus\core\modules\ComEditor\gui) manually, as discussed in this topic: [#311 (comment)](https://github.com/Arkensor/DayZCommunityOfflineMode/issues/311#issuecomment-1001046083)

nice, thx.

**Comment by 1435thunder** [comment]
Its normal, it only happens when using the mouse wheel to scroll up the list. 
To mitigate the issue, use the mouse and drag the scroll bar or type in the search bar at the top of the list!
